### ISSUE TITLE
pod CrashLoopBackOff on EKS v1.27.4 and v1.24.16-eks

### ISSUE THREAD

**Issue by infa-ddeore** [XenitAB/spegel#213]
spegel pods are in crashlookbackoff with below error

```
Defaulted container "registry" out of: registry, configuration (init)
{"level":"error","ts":1696142954.9523215,"caller":"build/main.go:72","msg":"","error":"Containerd registry config path is /etc/containerd/certs.d:/etc/docker/certs.d but needs to be /etc/containerd/certs.d for mirror configuration to take effect","stacktrace":"main.main\n\t/build/main.go:72\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:250"}
```

helm chart version used to install
```
helm upgrade --install --namespace kube-system --version v0.0.11 spegel oci://ghcr.io/xenitab/helm-charts/spegel
```

image: `ghcr.io/xenitab/spegel@sha256:8c73772fbf07b8b0458a26dc991b48f06b407082f3f554a0ee3bffd1ff89bf91`

containerd config file:
```
# cat /etc/containerd/config.toml
version = 2
root = "/var/lib/containerd"
state = "/run/containerd"

[grpc]
address = "/run/containerd/containerd.sock"

[plugins."io.containerd.grpc.v1.cri".containerd]
default_runtime_name = "runc"
discard_unpacked_layers = true

[plugins."io.containerd.grpc.v1.cri"]
sandbox_image = "602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.5"

[plugins."io.containerd.grpc.v1.cri".registry]
config_path = "/etc/containerd/certs.d:/etc/docker/certs.d"

[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
runtime_type = "io.containerd.runc.v2"

[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
SystemdCgroup = true

[plugins."io.containerd.grpc.v1.cri".cni]
bin_dir = "/opt/cni/bin"
conf_dir = "/etc/cni/net.d"
```

**Comment by infa-ddeore** [comment]
same issue would happen on `v1.24.16-eks` as it has below registry config path

```
[plugins."io.containerd.grpc.v1.cri".registry]
config_path = "/etc/containerd/certs.d:/etc/docker/certs.d"
```

**Comment by phillebaba** [comment]
This was fixed in #191 that was part of the v0.0.12 release. Could you please upgrade and try again?

**Comment by infa-ddeore** [comment]
`v0.0.12`, worked, thanks

readme still has v0.0.11 reference so i used it
